data/starts.txt

```diff
@@ -59,6 +70,128 @@ conversation "default intro"
+			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
+		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`
+	scene "scene/train"
+	`The train trip from your hometown to the capital is calm and without incident. Several seats forward, a young couple close to your age discuss the down payment on an apartment as their legs kick their luggages. Behind you is the only other person in the car; a much older man wearing greasy overalls, who you assume comes from off-world by how his hair has turned gray.`
+	`	Before long, you arrive in New Reykjavik and find the government office responsible for handling starship loans. The place is relatively empty compared to the photos you'd seen online a week prior; most applicants have already been processed in the week after exam results were posted. The only people left are those sick at the time, those too lazy to apply in a timely manner, and those willing to wait if it means avoiding a mosh pit of students in a government building. You hand over your application papers, and after verifying them against online records, the office worker hands you two cards. One of them is the card for applying for a loan at a nearby bank; the other is your pilot's license, fully paid for.`
+	`	The bank is only a few blocks away from the office. You insert the card into an automatic clerk, and after a soft ping, a large document fills its screen.`
+	`	As you scroll through, a synthesized voice says, "You are borrowing 580,000 credits to be repaid over the course of a year. Your daily interest rate is 0.2%, which means that your daily payments are 2,241 credits, and by the end of the year you will have paid 237,784 credits in interest. Please sign your name on the screen when you are ready."`
+	`	After carefully rereading every page of the loan agreement, you sign your name:`
+	name
+	`	Eager to get started, you rush out of the bank, hop on the metro, and head towards the shipyard. You can barely keep your face straight as the rest of the carriage wear blank expressions. After getting off, you consider throwing away your planetary transport pass, but decide against it.`
```

Comment:
This might be breaking the "don't tell me what I'm thinking rule."

```suggestion
	`	Eager to get started, you rush out of the bank, hop on the metro, and head towards the shipyard. You can barely keep a straight face as the rest of the carriage wear blank expressions. After getting off, you consider throwing away your planetary transport pass, but decide against it.`
```

Reply:
It's no more egregious than the description of the player's expression in the default start.